### ISSUE TITLE
No file directory

### ISSUE THREAD

**Issue by NKessell** [carykh/jumpcutter#66]
I am getting an error when I try to input the video I want to plug into program. 
the location of the video I am trying to plug in is where the program is and is in mp4 format. why would I be getting this error?

**Comment by pniedzwiedzinski** [comment]
Could you paste the error message?

**Comment by NKessell** [comment]
> Could you paste the error message?

![problems with input_LI](https://user-images.githubusercontent.com/39757242/56080819-d29b3c80-5e59-11e9-93fb-76007b4cc087.jpg)
![temp problems](https://user-images.githubusercontent.com/39757242/56080820-d929b400-5e59-11e9-9cc9-66dba905e7b2.PNG)

**Comment by pniedzwiedzinski** [comment]
After you run the command, do you have `TEMP` folder?

**Comment by NKessell** [comment]
> After you run the command, do you have `TEMP` folder?

yes that is what the second image is coming from

**Comment by pniedzwiedzinski** [comment]
It looks like ffmpeg didn't generate `audio.wav`. Check if `TEMP/audio.wav` exists. What version of windows do you use?

**Comment by NKessell** [comment]
> It looks like ffmpeg didn't generate `audio.wav`. Check if `TEMP/audio.wav` exists. What version of windows do you use?

windows 10, how would I check if it exists?

**Comment by pniedzwiedzinski** [comment]
go to the folder where `jumpcutter.py` is and there should be the `TEMP` folder

**Comment by NKessell** [comment]
i dont have a temp folder I dont think.

> go to the folder where `jumpcutter.py` is and there should be the `TEMP` folder

**Comment by pniedzwiedzinski** [comment]
yes, after error you should have `TEMP` folder and there should be results created with ffmpeg

**Comment by NKessell** [comment]
> yes, after error you should have `TEMP` folder and there should be results created with ffmpeg
okay yes, so how do I check if it has audio.wav from a text file?(params)

**Comment by NKessell** [comment]
> yes, after error you should have `TEMP` folder and there should be results created with ffmpeg

could issue #26 help with this problem?

**Comment by pniedzwiedzinski** [comment]
> > yes, after error you should have `TEMP` folder and there should be results created with ffmpeg
> 
> could issue #26 help with this problem?

No, this is a different problem

**Comment by pniedzwiedzinski** [comment]
You could try to insert an absolute path to `Lec1.mp4`

**Comment by NKessell** [comment]
> You could try to insert an absolute path to `Lec1.mp4`

I could do that however, would that mean I will have to do that with every video I try to pass through?

**Comment by pniedzwiedzinski** [comment]
maybe `./Lec1.mp4` will work

**Comment by NKessell** [comment]
> maybe `./Lec1.mp4` will work 
nope it didnt work. I am pretty confused as to why this is not working.

**Comment by NKessell** [comment]
> > maybe `./Lec1.mp4` will work
> > nope it didnt work. I am pretty confused as to why this is not working.

how would I inset an absolute path to lec1.mp4?

**Comment by pniedzwiedzinski** [comment]
In file explorer in properties, you have a path to the file. Just copy and paste

**Comment by NKessell** [comment]
> In file explorer in properties, you have a path to the file. Just copy and paste

paste into the environment variables like ffmpeg?

**Comment by NKessell** [comment]
> In file explorer in properties, you have a path to the file. Just copy and paste

I tried it and same result

**Comment by pniedzwiedzinski** [comment]
> > In file explorer in properties, you have a path to the file. Just copy and paste
> 
> paste into the environment variables like ffmpeg?

no, after  --input_file

**Comment by Ch4rlie97** [comment]
I had the exact same problem, and I managed to fix it. 

1. Make sure "Lec1" is in the same folder as the jumpcutter.py.

2. When you write on the console the input name, add the file extension .mp4 or the one you use so that it looks like this: "--input_file Lec1.mp4"

3. Do the same with the output: "--output_file name.mp4"

**Comment by SreenikethanI** [comment]
> 2. When you write on the console the input name, add the file extension .mp4 or the one you use so that it looks like this: "--input_file Lec1.mp4"
> 3. Do the same with the output: "--output_file name.mp4"

I guess this might be the solution...

**Comment by Leteen** [comment]
That still doesn't work for me

**Comment by NKessell** [comment]
> I had the exact same problem, and I managed to fix it.
> 
> 1. Make sure "Lec1" is in the same folder as the jumpcutter.py.
> 2. When you write on the console the input name, add the file extension .mp4 or the one you use so that it looks like this: "--input_file Lec1.mp4"
> 3. Do the same with the output: "--output_file name.mp4"

No I have tried that solution and that too doesnt seem to work either.

**Comment by Leteen** [comment]
You did install ffmpeg right?

**Comment by NKessell** [comment]
> You did install ffmpeg right?

I did yes and made a path to it in environmental variables

**Comment by Leteen** [comment]
it's an issue with that

**Comment by NKessell** [comment]
https://github.com/carykh/jumpcutter/issues/66#issuecomment-483074493
Okay what would be the problem as if it is that is hopefully an easy enough fix

**Comment by EfrenPy** [comment]
I had the same problem, got to solve it by navigating through the cmd window to the folder of the jumpcutter.py and the video files and running the script there.

**Comment by NKessell** [comment]
https://github.com/carykh/jumpcutter/issues/66#issuecomment-486153271
But that is what I have done as the jumpcutter.py in the folder with the video files. Im not sure what ypu are saying, could you elaborate

**Comment by EfrenPy** [comment]
What I mean is that in windows you can't right-click and ejecute in terminal, so you have to open terminal and navigate to the folder before running the jumpcutter.py. 
I was running it from the main folder of the terminal lke this: 
$ python C:\...\jumpcutter-master\jumpcutter.py  -input_file namefile.mp4
But this gave me that problem so now I do:
$ cd C:\...\jumpcutter-master
and
$ python jumpcutter.py -input_file namefile.mp4
This works for me.

**Comment by NKessell** [comment]
Okay I will give it a go

**Comment by Ch4rlie97** [comment]
> 
> 
> What I mean is that in windows you can't right-click and ejecute in terminal, so you have to open terminal and navigate to the folder before running the jumpcutter.py.
> I was running it from the main folder of the terminal lke this:
> $ python C:...\jumpcutter-master\jumpcutter.py -input_file C:...\jumpcutter-master\namefile.mp4
> But this gave me that problem so now I do:
> $ cd C:...\jumpcutter-master
> and
> $ python jumpcutter.py -input_file namefile.mp4
> This works for me.

you can just open the folder you want and click where the folder location is written, then just write "cmd" and presss enter to open the terminal from there

**Comment by sonicwolfsdutch** [comment]
I just had to install the requirements.txt  i did this [yt tutorial](https://youtu.be/pHR3ttH5t-w) to install ffmpeg for win 10.
After that it still dint work i restarted my computer and it almost worked i had to disabled my antivirus after that it worked fine! 

Also i updated to 3.7 and updated pip  and you have to select that python will work with the cmd.
